src/styles/Blocks/Stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url(../images/star-active.svg) no-repeat center;
+    width: 16px;
+    height: 16px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+
+    &:last-child {
+      background: url(../images/star.svg) no-repeat center;
+    }
+  }
+}
```

Comment:
Your stars should work same as you did before [here](https://github.com/AnnaNazarova23/layout_stars/tree/develop)